lib/model/query/entities.js

```diff
@@ -96,7 +109,7 @@ const processSubmissionEvent = (event) => (container) =>
     .then((entity) => {
       if (entity != null) {
         return container.Audits.log({ id: event.actorId }, 'entity.create', { acteeId: entity.aux.dataset.acteeId },
-          { entity: { uuid: entity.uuid, label: entity.def.label, dataset: entity.aux.dataset.name },
+          { entity: { uuid: entity.uuid, label: entity.aux.currentVersion.label, dataset: entity.aux.dataset.name },
```

Comment:
This entity creation path (via submissions) seems less tied to users and permissions than other paths. It feels like it would be more mysterious if someone approved a submission, and no entity was made when everything else (dataset, form design) pointed to creating an entity. I don't think we'd stop them from approving the submission, and i also don't think we'd surface the permission error, so it would be hard to track down that it was a permissions problem.